packages/contracts/test/diamond/facets/BondingCurveFacet.t.sol

```diff
@@ -72,7 +74,15 @@ contract ZeroStateStakingTest is BondingCurveFacetTest {
         );
         vm.stopPrank();
 
+        uint256 tokReturned = IBondingCurveFacet.purchaseTargetAmountFromZero(
```

Comment:
these tests are fine but we also need
- another test that will use purchaseTargetAmount
- other tests to make sure  purchaseTargetAmountFromZero and purchaseTargetAmount returns correct calculations

Reply:
I made some corrections

Reply:
Please point me where the test calling purchaseTargetAmount is ?
and here we only assess that tokreturned is what getshare returns but we need to assess that it is equal to an expected value that we need to calculate here with  _supply * ((1 + _tokensDeposited / _connectorBalance) ^ (_connectorWeight / 1000000) - 1)

Reply:
It has a seperate function. Currently line 189. Thanks for the review. Updated.